spk/ffsync/src/wizard/uninstall_uifile.sh

```diff
@@ -51,6 +48,7 @@ PAGE_FFSYNC_REMOVE=$(/bin/cat<<EOF
 		"subitems": [{
 			"key": "wizard_dbexport_path",
 			"desc": "Database export location",
+			"emptyText": "${SYNOPKG_PKGDEST_VOL}/${SYNOPKG_PKGNAME}/backup",
```

Comment:
use "defaultValue" (same as above)

Reply:
I want to use "emptyText" and not "defaultValue" because the example may not work as-is in the user's specific environment. I want the user to explicitly consider their correct value.

Reply:
so leave it, it is worth nothing